tests/functional/Tribe/Events/Importer/File_Importer_Events_BooleanFieldsTest.php

```diff
@@ -252,4 +252,40 @@ public function it_should_accept_falsy_values_to_show_map( $falsy_boolean_value
 		$this->assertEquals( '', get_post_meta( $post_id, '_EventShowMapLink', true ) );
 	}
 
+	public function currency_positions() {
+		return [
+			[ 'prefix', 'prefix' ],
+			[ 'prefix', 'Prefix' ],
+			[ 'prefix', 'PREFIX' ],
+			[ 'postfix', 'postfix' ],
+			[ 'postfix', 'Postfix' ],
+			[ 'postfix', 'POSTFIX' ],
+			[ 'prefix', 'before' ],
+			[ 'prefix', 'Before' ],
+			[ 'prefix', 'BEFORE' ],
+			[ 'postfix', 'after' ],
+			[ 'postfix', 'After' ],
+			[ 'postfix', 'AFTER' ],
```

Comment:
In the UI, currency position uses the word `suffix`. I think that should be represented here and maybe replace `postfix`?